apps/framework-docs/src/pages/moose/moose-cli.mdx

```diff
@@ -154,6 +154,19 @@ moose generate migration --url https://<remote-env> --token <api-token> --save
 - Omit `--save` to output to stdout without writing files
 - Requires `[features].olap = true`; execution of saved plan requires `[features].ddl_plan = true`
 
+### DB Pull (External Tables)
+Refresh `EXTERNALLY_MANAGED` table definitions from a remote ClickHouse instance.
+```bash
+moose db pull --connection-string <CONNECTION_STRING> [--file-path <OUTPUT_PATH>]
+```
+- `--connection-string`: ClickHouse URL; native `clickhouse://` is auto-converted to HTTP(S). Include `?database=` or the CLI will query the current database.
```

Comment:
give an example connection string? we've seen that this can be pretty tricky to get exactly right, no?